design/ceph/osd-migration.md

```diff
@@ -36,139 +25,107 @@ considered in the future:
   and application data on slower media.
 - Setups with multiple OSDs per drive, though with recent Ceph releases the
   motivation for deploying this way is mostly obviated.
+- Mix of encrypted and unencrypted OSDs in the cluster.
```

Comment:
Since the encrypted setting is on a `storageClassDeviceSet`, is there any reason not to support a mix of encrypted/unencrypted OSDs as long as they are in different device sets? I just heard someone asking for this scenario a couple days ago. In that case, this migration feature may need to only migrate a subset of OSDs. Basically, we would need to migrate whichever OSDs don't match their desired settings.

Reply:
That looks like something that can be done. Removing it from the non-goals.